README.md

```diff
@@ -380,7 +380,46 @@ Staltz can see Arj has invited Mix because he's replicating Arj's
 
 ## 5. Security Considerations
 
-TBD
+### 5.1. Degraded confidentiality at scale
+
+As noted in the [private-groups-original-notes], SSB Private Groups are based on
+symmetric encryption and static keys (no key rotation), which means it cannot
+guarantee [perfect-forward-secrecy] neither [post-compromise-security]. This
+makes SSB Private Groups less likely to guarantee confidentiality at scale, when
+a group has dozens or hundreds of members. If any of the members leaks the group
+secret, all past communication in the group is compromised.
+
+This also means that any group member can add new members to the group. They may
+even do so without publishing a `group/add-member` message, which effectively
+allows any group member to invite a third party that will remain undetected by
+existing group members.
+
+SSB Private Groups with several members should thus not be considered
+confidential nor forward secret, and should be used with caution. Our security
+model assumes that the group is as trustworthy as the _least_ trustworthy member
+in the group.
+
+The choice of symmetric encryption (as opposed to, e.g. hash-ratchet or
+double-ratchet) was made because of efficiency. Symmetric encryption means
+adding new members or publishing new content is `O(1)` fast, and the system is
+overall simple to implement.
+
+### 5.2. Membership analysis attacks
+
+A group feed is discoverable only by the group members, because its announcement
+is encrypted. However, an eavesdropper who can replicate a peer's metafeed tree
+will notice the presence of these encrypted message on shard feeds. If the
+eavesdropper can do that for several peers, then they may perform an analysis
+attack where they try to correlate the encrypted messages on the shard feeds to
+the group members.
+
+This can be mitigated by publishing dummy encrypted messages on the shard feed
+at random intervals, with the downside of increasing the size of the shard feed,
+and thus making partial replication heavier.
+
+It is currently unknown how much can be learned from such an analysis attack, as
+it depends on the network's complexity, such as number of peers, number of
+groups, and number of relationships between peers.
```

Comment:
I think it is worth adding a section on rooms as opposed to pubs. Because this changes the model from connect to trusted peers instead of semi-trusted (pubs).

Reply:
Hmm, what do you mean, why do we need to talk about pubs/rooms here? (Assume we do e.g. hops 2 or 3 replication, so that A replicating B may never connect A-B via a room)

Reply:
Now that I think about it, it's probably too off-topic for here. But I was thinking of a person that briefly knew about SSB and wanted to read this spec about group encryption and might think this is a bigger problem that is might be in practice because we have for a large part moved away from relying on pubs. Most pubs was not properly protected so it was easy to do a full network scans basically. Anyway lets leave that out of this spec.

Reply:
Ah right, you're thinking about an eavesdropper on an open pub for instance.